opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractClientTest.java

```diff
@@ -56,14 +65,15 @@ protected void initTracing(ServletContextHandler context) {
     public void testDefaultConfiguration() {
         MockTracer mockTracer = new MockTracer(new ThreadLocalScopeManager(), Propagator.TEXT_MAP);
```

Comment:
Not related to this PR, but could simplify this to just use default constructor.